### ISSUE TITLE
Purpose of the macro...

### ISSUE THREAD

**Issue by Razorwynd** [EinBaum/SP_SwingTimer#6]
Maybe you could add a brief explanation of what the macro is supposed to accomplish in the readme.txt 

Because right now I am a bit confused. From what I can tell, once the slam cast finishes then the auto attack timer starts again. So, to me, it seems optimal to start the slam cast immediately following an auto attack. Furthermore, it seems like your macro is a very non-ideal time to start the slam cast because you have just progressed through 2 seconds of the swing timer.  

Given that this is the case then I might offer two suggestions:

First make the swing timer count up, rather than down... the reason for this is that the start value (or maximum duration) of the swing timer is variable and always changing due to flurry. 

The first suggestion would then allow you to make a spamable macro that only starts the slam cast in the first 0.1 seconds of your swing timer, thereby minimizing the amount of wasted due the the reset of the swing timer resulting from casting slam.

Hopefully this wall of text makes sense!

**Comment by EinBaum** [comment]
Thanks for the feedback. I can understand the confusion. Back when I wrote this AddOn, I've created it for myself, thinking it wouldn't become so popular :laughing: 

The macro was used on the Emerald Dream server to abuse a bug: If you start Slam when the swing timer is below the casting time of Slam, both autoattack and Slam would hit the target. This is most likely fixed on other servers.

I'll add a note or remove the macro.

I don't really want to change how the swing timer works right now. I'm playing a caster class right now and it's not interesting for me to re-do everything.

Here's a macro to cast Slam only in the first 0.x seconds:

```
/run if false then CastSpellByName"Slam()" end
/run if not _a then for i=1,72 do if IsAttackAction(i) then _a=i end end end if not IsCurrentAction(_a)then UseAction(_a)end if st_timer>UnitAttackSpeed"player"*0.9 then CastSpellByName"Slam()" end

```
